src/HashBus.Projector.MostRetweeted/MostRetweetedProjection.cs

```diff
@@ -19,17 +21,17 @@ public MostRetweetedProjection(IRepository<string, IEnumerable<Retweetee>> retwe
             this.retweetees = retweetees;
         }
 
-        public void Handle(Application.Events.TweetAnalyzed message)
+        public Task Handle(TweetAnalyzed message, IMessageHandlerContext context)
         {
             if (message.Tweet.RetweetedTweet == null)
             {
-                return;
+                return Task.FromResult(0);
             }
 
             var trackRetweetees = this.retweetees.GetAsync(message.Tweet.Track).GetAwaiter().GetResult().ToList();

```

Comment:
make the Handle method `async` and remove `.GetAwaiter().GetResult()`, making it:

```
( await this.mentions.GetAsync(message.Tweet.Track) ).ToList();
```

then you can remove all the `return Task.FromResult(0);`